src/components/chip/themes/dark/chip.indigo.scss

```diff
@@ -17,23 +17,43 @@ $theme: $indigo;
 
 :host([variant='success']) button,
 :host([selected][variant='success']) button:not([disabled]) {
+    background: color(success, 800);
+
     &:focus {
-        box-shadow: 0 0 0 rem(3px) color(success, 800);
+        box-shadow: 0 0 0 rem(3px) color(success, 900);
+    }
+
+    &:hover {
+        background: color(success, 700);
     }
 }
 
 :host([variant='warning']) button,
 :host([selected][variant='warning']) button:not([disabled]) {
+    color: color(gray, 50);
+
     &:focus {
+        color: color(gray, 50);
         box-shadow: 0 0 0 rem(3px) color(warn, 900);
     }
+
+    &:hover {
+        color: color(gray, 50);
+    }
 }
 
 :host([variant='danger']) button,
 :host([selected][variant='danger']) button:not([disabled]) {
+    background: color(error, 600);
```

Comment:
It seems that the danger button styles you're updating are the same for light and dark themes, you can export them to `shared`. The same applies for the success backgrounds.